ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -22,15 +22,15 @@
       data-cmp-is="adaptiveFormContainer"
       data-cmp-context-path="${request.contextPath}"
       data-cmp-path="${resource.path}"
-      class="cmp-adaptiveform-container ${grid.cssClass} cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}">
```

Comment:
@rismehta Why were we doing this?
`cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}`

Reply:
We don't need the CSS class in edit mode

Reply:
well we don't need this at all.